### DISCUSSION TITLE
~/Screenshots? Best way to specify different location?

### DISCUSSION THREAD

**Discussion by eterps** [Manjaro-Sway/manjaro-sway#416] [Q&A] [answered]
Apparently it has been decided recently that there should be a folder called 'Screenshots' in my home dir, removing it just creates a new one.
What is the best way to specify a different location for it?

**Comment by boredland** [comment, marked as answer]
hi there! adding a `XDG_SCREENSHOTS_DIR=` to your `~/.config/user-dirs.dirs` should be enough. Let me know if that worked, so we could add that hint to our support docs :wink:

**Reply by eterps** [reply]
Yeah, that was my assumption, but it didn't work. I suspect it didn't because I am using fish as my shell.

**Reply by boredland** [reply]
well, you could export that variable and give it a try.

**Reply by boredland** [reply]
Nevermind, user dirs are [sourced](https://github.com/manjaro-sway/desktop-settings/blob/sway/community/sway/etc/skel/.profile#L31) in `.profile`, so your shell shouldn't matter.

**Reply by eterps** [reply]
That doesn't work either. I can see the exported var just fine in my shell, but I assume the problem is that the screenshot process doesn't get to see that var.

**Reply by boredland** [reply]
does your `.profile` contain that `source $HOME/.config/user-dirs.dirs` line? because afaik by convention that should have been sourced before anything else on sway start.

**Reply by eterps** [reply]
Thank you, that seemed to be the problem. It works now 🙏

**Reply by boredland** [reply]
[Added](https://github.com/manjaro-sway/manjaro-sway/commit/df5c176362c5de9988f59bcf24bbd61b8133fe06) to support.md
